core/domain/wipeout_service_test.py

```diff
@@ -4423,3 +4430,147 @@ def test_verify_user_delete_when_profile_user_not_deleted_is_false(self):
             wipeout_service.get_pending_deletion_request(self.profile_user_id))
         self.assertTrue(
             wipeout_service.verify_user_deleted(self.profile_user_id))
+
+
+class FirebaseAccountWipeoutTests(test_utils.GenericTestBase):
```

Comment:
But the point is, this test shouldn't bypass core/platform/models.py and depend on a specific platform-layer implementation. There should be generic functions in both platform layers that this test works on, so that the tests pass both now and whenever the switch in which platform layer implementation to use is made.

Reply:
Hmm, but that is what I do? In wipeout_services, I call "auth_services.delete_associations". I'm not sure what changes you're suggestion need to be made here?

Reply:
But this test refers to "firebase" all over the place, and it shouldn't do so if it's platform-impl independent?

Reply:
Ah the goal here is to make sure the Firebase accounts are really deleted when the Firebase platform is being used. That's precisely why I don't think it belongs in wipeout, because it's something that should only happen when Firebase is the platform-layer (hence it should be in the Firebase unit tests, not wipeout test).

Reply:
Why not have both implementations expose an are_all_auth_associations_deleted() function, and use that to verify?

Reply:
Obsolete, all of this has moved into firebase_auth_services_test.